features/dd-sdk-android-trace/src/main/java/com/datadog/opentracing/PendingTrace.java

```diff
@@ -84,9 +89,27 @@ public long getCurrentTimeNano() {
 
   public void registerSpan(final DDSpan span) {
     if (traceId == null || span.context() == null) {
+      internalLogger.log(
+        InternalLogger.Level.ERROR,
```

Comment:
In `DDSpan` when we are dropping the span `warn` level is used, but here it is an `error` in the whole file. Should they be aligned?

Reply:
In DDSpan, we use Warning as the cause of the drop is (most likely) misuse of the SDK, usually by calling `finish()` more than once.
Here the drop reason are ones that _should not happen™️_, hence the use of Error level.